ios/brave-ios/Sources/Brave/WebFilters/CustomFilterListStorage.swift

```diff
@@ -148,37 +154,31 @@ import WebKit
     }
 
     let fileURL = try customRulesFileURL()
```

Comment:
the `customRulesFileURL` use to get/create the folder. Now it just gets. This results in an error when saving. You should either put back the getOrCreate in the above method or change this to lazily create the folder first by calling 
`getOrCreateCustomRulesFolder` first

Note: my bad for the poor naming of the `customRulesFileURL` function.

Reply:
Good catch, will add a call to `getOrCreateCustomRulesFolder` above the write

Reply:
Fixed here: https://github.com/brave/brave-core/pull/24555/commits/f49166579f840b734f992057e688a83fb2ee344a